src/app/scenes/app.tsx

```diff
@@ -20,19 +22,23 @@ const setLanguage = useLanguage(
       ...require('@assets/i18n/en.json'),
       ...require('../../libs/auth/assets/i18n/en.json'),
```

Comment:
As discussed, let's keep translation separately from assets and outside the `libs` directory. Please move  this file into `src/i18n/auth`.

Could you please apply the same change of other translation files?